### ISSUE TITLE
Creating an empty file

### ISSUE THREAD

**Issue by Vsd1106** [nanoporetech/dorado#249]
>dorado basecaller /Users/Vlad/Desktop/Dorado/bin/rna003_120bps_sup@v3 pod5s > calls.bam 
I am running this command and it only generates an empty calls.bam file. 
the output is:[2023-06-19 10:19:27.363] [info] > Creating basecall pipeline 
After 15 seconds I can input a new command and I'm left with a 0kb file.

**Comment by vellamike** [comment]
Hi @Vsd1106 ,

What is the content of the directory called  `pod5s` ? Are you certain there are valid pod5 files in there?

**Comment by Vsd1106** [comment]
It contains one pod5 file that was converted from a fast5 file. The file was downloaded from a public database as we don’t have any of our own data at the moment.

**Comment by vellamike** [comment]
Thank you, can you copy and paste the full output of dorado? Is this running on a Mac?

**Comment by Vsd1106** [comment]
No it's running on widows
the output is:
[2023-06-19 13:46:25.712] [info] > Creating basecall pipeline
C:\Users\Vlad\Desktop\Dorado\Software>
Renamed the bin file so it would be more suggestive

**Comment by vellamike** [comment]
I don't think I fully follow, what do you mean when you say "Renamed the bin file so it would be more suggestive" ? I don't see a bin file in the output?

**Comment by vellamike** [comment]
Follow up question: Which command line terminal environment are you using on windows?

**Comment by Vsd1106** [comment]
I fixed it I changed the batchsize and it worked. Don't know why I didn't get a message in regards to memory usage.

**Comment by tijyojwad** [comment]
so the issue was the default batch size was too large for your GPU memory but dorado just exits without a CUDA OOM error? We can add a more helpful error message

**Comment by DCossey** [comment]
I am getting a similar problem. I have a database of pod5 files and run the command as follows (on Ubuntu22): 
/home/tanentzap/dorado-0.5.0-linux-x64/bin/dorado duplex --batchsize 1 sup /var/lib/minknow/data/Test_try4_no_AS_Dan/Test_try4_Dan/20240126_2001_P2S-01237-A_PAO25658_72f55b01/pod5  > test_try4.bam

I set the batchsize to 1 because previously I got the error "out of memory" and this fixed it. However our computer definitely has enough memory so I am not sure why we got that error previously. Anyway, now I have another problem, namely, this command ran for 24 hours and produced no errors, but the output file had 0 bytes and nothing in it. What is very strange is that two days ago i did the same command on a different, much smaller, dataset and I got my bam file just fine. So I do not understand what the problem is now. Any help would be much appreciated.
